src/fsharp/FSharp.Core/printf.fs

```diff
@@ -1004,7 +1007,7 @@ module internal PrintfImpl =
 
         let verifyMethodInfoWasTaken (_mi : System.Reflection.MemberInfo) =
```

Comment:
@forki.  
Since we are doing cleanup can we rename _m back to m, and add ignore m in the # else code path.